src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -53,13 +54,17 @@ html {
             box-shadow: 2px 2px #7a766d;
             display: inline-block;
             padding: 1em;
-            min-height: 9.6em;
-            width: 25em;
+            min-width: 25em;
             background-color: #fff;
-            margin-right: 10px;
             white-space: normal;
-            vertical-align: middle;
+            display: table-cell;
         }
+        
+        // override inline label style for panel description 
+        .fl-prefsEditor-display-block {
```

Comment:
Great idea, I added the styling to the on/off toggle instead which will make it easier to scale in the future as well